apps/ui/src/components/trace-viewer/Span.tsx

```diff
@@ -47,11 +47,11 @@ export function Span({
             {Number(value).toFixed(2)} {unit}
           </span>
         </p>
-        <div className={`absolute h-2 bg-graphiql-border w-full`}></div>
+        <div className={`absolute h-2 bg-neutral/30 rounded-md w-full`}></div>
```

Comment:
```suggestion
        <div className={`absolute h-2 bg-neutral-100 rounded-md w-full`}></div>
```

I dont see a `neutral/30` color, am I missing something?